charts/verdaccio/values.yaml

```diff
@@ -54,8 +54,13 @@ ingress:
 
 ## Service account
 serviceAccount:
+  # Specifies whether a service account should be created
   enabled: false
-  # name:
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
```

Comment:
```suggestion
  # If not set and enabled is true, a name is generated using the Chart's fullname template
```

Reply:
If this isn't what you were trying to convey, feel free to reject this suggestion.